### PR TITLE
osd: add possibility to exclude device in osd_auto_discovery

### PR DESCRIPTION
**Pull request by guits**
Add a new `osd_auto_discovery_exclude` to give the possibility of
excluding some devices in auto_discovery scenario.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: roles/ceph-osd/defaults/main.yml [hunk 1]
```diff
@@ -43,6 +43,9 @@ devices: []
 #
 osd_auto_discovery: false
 
+# Any device containing these pattern in their path will be excluded.
```
Comment (leseb) [inline roles/ceph-osd/defaults/main.yml]:
```suggestion
# Any device containing these patterns in their path will be excluded.
```

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

**Pull request merged**